Rationale.md

```diff
@@ -240,3 +240,20 @@ architectures there may be a need to revisit some of the decisions:
 * When different languages have different expectations then it's unfortunate if
   WebAssembly measurably penalizes one's performance by enforcing determinism
   which that language doesn't care about, but which another language may want.
+
+
+## Feature Testing - Motivating Scenarios
```

Comment:
"Motivating Scenarios for Feature Testing"? Different Markdown parsers treat dashes differently (should be em dash too :p)